src/tests/admin/conferenceroomoutoforderplanning/OutOfOrderIsCreatedAndEnabled.java

```diff
@@ -0,0 +1,67 @@
+package tests.admin.conferenceroomoutoforderplanning;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.json.JSONException;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomOutOfOrderPlanningPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC01: Verify an Out Of Order can be created and enabled in Out Of Order Planning
+ * TC28: Verify that when an Out Of Order is created and enabled a message that says 
+ * "Out of order was created successfully." is displayed
+ * @author Yesica Acha
+ *
+ */
+public class OutOfOrderIsCreatedAndEnabled {
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> testData = excelReader.getMapValues("OutOfOrderPlanning");
+	String roomName = testData.get(4).get("Room Name");
+	String startDate = testData.get(4).get("Start date");
```

Comment:
As well here, keep only roomName and title as global variables, others declare inside the test method